extensions/amp-analytics/0.1/iframe-transport.js

```diff
@@ -147,6 +160,48 @@ export class IframeTransport {
         '/$internalRuntimeVersion$/iframe-transport-client-v0.js';
   }
 
+  /**
+   * Uses the Long Task API to create an observer for when 3p vendor frames
+   * take more than 50ms of continuous CPU time.
+   * Currently the only action in response to that is to log. It will log
+   * once per LONG_TASK_REPORTING_THRESHOLD_ that a long task occurs. (This
+   * implies that there is a grace period for the first
+   * LONG_TASK_REPORTING_THRESHOLD_-1 occurrences.)
+   * @VisibleForTesting
+   * @private
+   */
+  createLongTaskObserver_() {
+    if (!isLongTaskApiSupported(this.ampWin_)) {
+      return;
+    }
+    // TODO(jonkeller): Consider merging with jank-meter.js
+    this.longTaskObserver_ = new this.ampWin_.PerformanceObserver(entryList => {
+      if (!entryList) {
+        return;
+      }
+      entryList.getEntries().forEach(entry => {
+        if (entry && entry['entryType'] == 'longtask' &&
+            (entry['name'] == 'cross-origin-descendant' ||
+            (getMode().localDev &&
```

Comment:
FYI: You can do `ad.localhost:8000` to create cross origin environment. Please test with that and see if we can remove code here.

Reply:
Which code are you hoping to remove? I don't think we can remove the check for cross-origin-descendant, as that is definitely getting hit:
![image](https://user-images.githubusercontent.com/2455047/33029140-ce041c74-cde5-11e7-8224-d3e88189ef70.png)

Reply:
I am hoping to remove the special check for `(getMode().localDev && ...)` here since we are able to mock `cross-origin-descendant` in local development mode.